pyOCD/tools/pyocd.py

```diff
@@ -545,7 +545,7 @@ def do_read(self, args):
         addr = self.convert_value(args[0])
         if len(args) < 2:
             count = 4
-                else:
+        else:
```

Comment:
I think there may be some other errors in pyocd.py from another recent change to `pyocd-tool`.